resources/less/hakija.less

```diff
@@ -1027,11 +1047,16 @@ textarea.application__form-text-area__size-large {
   font-size: 14px;
 }
 
-.application__form-checkbox:checked + label {
+label.application__form-checkbox.checked {
```

Comment:
Laita nämä sisäkkäin ylemmän määrittelyn kanssa käyttäen .checked

Sama myös disabledille